development_helpers/docker-compose.dev.yml

```diff
@@ -31,10 +33,31 @@ services:
       # - ../:/opt/CTFd:ro
     depends_on:
       - db
+      - llm_router
+      - cache
     networks:
         default:
         internal:
 
+  llm_router:
+    image: aivillage/llm_router:dev
+    user: root
+    restart: always
+    volumes:
+      - ../mock.json:/opt/models/mock.json
+    environment:
+      - REDIS_URL=redis://cache:6379
+    depends_on:
+      - cache
+    networks:
+        internal:
+    healthcheck:
```

Comment:
Adding health checks is a great idea. I probably should add one to ctfd_llmv as well